packages/prisma/schema.prisma

```diff
@@ -399,14 +410,15 @@ model Impersonations {
 }
 
 model ApiKey {
-  id         String    @id @unique @default(cuid())
+  id         String     @id @unique @default(cuid())
   userId     Int
   note       String?
-  createdAt  DateTime  @default(now())
+  createdAt  DateTime   @default(now())
   expiresAt  DateTime?
   lastUsedAt DateTime?
-  hashedKey  String    @unique()
-  user       User?     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  hashedKey  String     @unique()
+  apiKeyType ApiKeyType @default(OTHER)
```

Comment:
BTW we now have an `App` model. It theory we could just add a relationship like: `appId: "zapier"` that way we could have a way to fetch only zapier related api keys without having add a new type.